stubs/django/db/models/fields/__init__.pyi

```diff
@@ -158,3 +161,17 @@ def NullBooleanField(
     validators: Sequence[Callable[[Optional[bool]], None]] = ...,
     error_messages: Mapping[str, str] = ...,
 ) -> Optional[bool]: ...
+
+Field = Union[
```

Comment:
So I was naive thinking that this was going to work. These are functions, not types, and `pyre` isn't smart enough to be able to say "I want this to accept the union of the return value of these funcitons".

I think we're going to have to do a refactor and fix the fact that this whole stub file is a lie. These functions really should be types. I don't want to put that on you though. How about you just make `Field` an alias for `Any`, and I'll pick up the refactor once we've landed this.

```suggestion
Field = Any
```